src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -931,6 +941,10 @@ internal static bool CanBePassedByReference(BoundExpression expr)
                         _ => throw ExceptionUtilities.UnexpectedValue(patternIndexer.PatternSymbol)
                     };
                     return refKind != RefKind.None;
+
+                case BoundKind.Conversion:
+                    var conversion = ((BoundConversion)expr);
+                    return expr is BoundConversion { Conversion: { IsInterpolatedStringHandler: true }, Type: { IsValueType: true } };
```

Comment:
>return expr is BoundConversion { Conversion: { IsInterpolatedStringHandler: true }, Type: { IsValueType: true } }; [](http://example.com/codeflow?start=20&length=114)

Is the actual local introduced by ILGen layer or elswhere in the LocalRewriter?

Reply:
I mean the local that is passed by ref at the end. I just realized that it is probably the builder local that we are always creating. So we are just pretending that there is a ref modifier for the corresponding argument. Correct?

Reply:
> So we are just pretending that there is a ref modifier for the corresponding argument

That is correct.